### ISSUE TITLE
Accessibility: Android Talkback does not work properly anymore using the touchscreen

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-android#621]
### Description

Since the recent updates, Android Talkback does not seem to work correctly or how it used to work. I've tested it out, too, and navigating is a lot trickier than it used to be. You can't navigate with the touch screen and have to use gestures, I don't recall that happening in the past. 

### Step-by-step reproduction instructions

1. Enable Android Talkback as explained here: https://support.google.com/accessibility/android/answer/6007100?hl=en
2. Open Pocket Casts
3. Try to use Talkback to read out the names of podcasts etc. Make sure to use the touch screen to navigate by double tapping on items, it should not make any difference. The only way to navigate is to use the swipe left/right on the edge of the screen gestures. 
4. Try another app like WooCommerce Mobile, and you should notice how it supposed to work on double touch etc. 

### Screenshots or screen recording

https://user-images.githubusercontent.com/16253818/204998741-a03de1fa-86f9-40a7-97c5-cd010291bb1e.mp4

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Samsung S21 FE, Android 12, Pocket Casts v7.27-rc-3

5739015-zd-woothemes

**Comment by joashrajin** [comment]
Update from the user: 

> I took this problem to the BlindAndroidUsers list, and their solution was to delete storage. This worked instantly, and worked again with an Android update that happened yesterday. 

They mentioned that this issue seems to be specific to Pocket Casts

**Comment by azollafox** [comment]
One of our users is still having issue with TalkBack in the settings. They are on a Samsung Galaxy Z fold 3 which is running Android 13 and installed the beta version, which shows as 7.29 rc2 

5804428-zen

**Comment by joashrajin** [comment]
A new report of issues relating to this, @ashiagr 


This issue can be reproduced with the following steps.

1. With TalkBack on, open Pocketcasts.

2. When tapping any of the tabs at the bottom, and then trying to move around the screen, nothing will happen, and the app will have to be restarted, or the screen locked and unlocked, in order to fix this issue. It is also worth noting that although the tabs are there, TalkBack will not announce them.


5979407-zd-woothemes

**Comment by ashiagr** [comment]
@joashrajin, can you ask the user to try `7.33` open beta release? Let me know if it still exists in that release and I'll prioritize it. I was able to reproduce the issue and [fixed](https://github.com/Automattic/pocket-casts-android/pull/769/commits/dbaee2263b22230282b05683c04329855e603ffd) it around two weeks ago.

**Comment by joashrajin** [comment]
@ashiagr They confirmed that it fixed the problem 🎉

**Comment by leandroalonso** [comment]
They recently mentioned this:

> Encountering another potential issue, when opening the 
public version, 7.32 for the first time and clicking the sign in 
button, the app will crash, making the screen reader restart. Not 
sure, but I think I remember the sign up button doing the same thing 
as well.

**Comment by joashrajin** [comment]
@ashiagr We have a new case of Talkback issues, and the beta didn't fix the issue: 

Steps to replicate: 

- Enable the TalkBack screen reader in the Accessibility section of the system Settings app.
- Open Pocket Casts.
- Select the Profile tab.
- Select the Settings button.

At this point, the app completely freezes.

They are using the latest version of the app on a Samsung Galaxy S23 running One UI 5.1.

If a new report should be made for this, please let me know :)

**Comment by mchowning** [comment]
I'll close this issue because it seems like the original issue was fixed [here](https://github.com/Automattic/pocket-casts-android/pull/630) and a second issue that came up in this issue was fixed [here](https://github.com/Automattic/pocket-casts-android/pull/769/commits/dbaee2263b22230282b05683c04329855e603ffd). 

Also, it seems like [the report of TalkBack freezing on the settings screen](https://github.com/Automattic/pocket-casts-android/issues/621#issuecomment-1475905976) is likely not a current issue because we've gotten no new reports of it and I cannot reproduce it with my Samsung test device. If there are any remaining Talkback issues that come up, let's open new issues.
